src/databaseTypesFile.ts

```diff
@@ -942,7 +945,7 @@ export type Database = {
         }
         Returns: undefined
       }
-      insertClientAndTheirFamily: {
+      insert_client_and_family: {
```

Comment:
Why do we need to rename this?

Reply:
Sorry just following supabase convention but I can change it back

Reply:
Ah nice, fingers crossed it doesn't cause any issues for anyone